metadata.yaml

```diff
@@ -92,6 +92,12 @@ spec:
         description: The list of the Bigquery push subscriptions.
         varType: "list(object({\r\n    name                       = string,\r\n    table                      = string,\r\n    use_topic_schema           = optional(bool),\r\n    use_table_schema           = optional(bool),\r\n    write_metadata             = optional(bool),\r\n    drop_unknown_fields        = optional(bool),\r\n    ack_deadline_seconds       = optional(number),\r\n    retain_acked_messages      = optional(bool),\r\n    message_retention_duration = optional(string),\r\n    enable_message_ordering    = optional(bool),\r\n    expiration_policy          = optional(string),\r\n    filter                     = optional(string),\r\n    dead_letter_topic          = optional(string),\r\n    max_delivery_attempts      = optional(number),\r\n    maximum_backoff            = optional(string),\r\n    minimum_backoff            = optional(string)\r\n  }))"
         defaultValue: []
+        connections:
+          - source:
+              source: github.com/terraform-google-modules/terraform-google-bigquery
+              version: ">= 10.1.0"
+            spec:
+              outputExpr: "{ \"name\": table_ids[0], \"table\": \"${var.project_id}:${env_vars.BIGQUERY_DATASET}.${table_ids[0]}\"}"
```

Comment:
why are we using `var.project_id`? How would this expression be resolved in ADC?